mbuild/packing.py

```diff
@@ -57,9 +57,14 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2,
              seed=12345, edge=0.2, compound_ratio=None,
              aspect_ratio=None, fix_orientation=False, temp_file=None,
              update_port_locations=False):
-    """Fill a box with a compound using packmol.
+    """Fill a box with a `mbuild.compound` or `Compound`s using PACKMOL.
+
+   `fill_box` will take a single `mbuild.Compound` or a,
+   list of `mbuild.Compound`'s and return an `mbuild.Compound` that has
```

Comment:
Might be personal preference but I don't like use apostrophes in plurals, even if it clobbers an `s` on top of an acronym or quoted term